src/index.html

```diff
@@ -3,11 +3,393 @@
+            >
+            <img
+              src="./images/photos/6.jpg"
+              alt="Faena House photo 2"
+              class="gallery__photo"
+            >
+          </div>
+          <p class="gallery__object-descr">
+            <big class="bright-emphasis">Faena House </big>/ North Miami Beach, FL 33164
+          </p>
+        </section>
+
+        <input
+          type="checkbox"
+          id="photosHider"
+          class="gallery__checkbox"
+        >
+        <div class="gallery__hided">
+          <!-- VERY IMPORTANT!!! in scss variables change additionalPhotosCount-->
+          <section class="gallery__object">
+            <div class="gallery__images-row gallery__images-row--1-1">
+              <img
+                src="https://retsimages.s3.amazonaws.com/29/A10388229_1.jpg"
+                alt="Faena House photo 1"
+                class="gallery__photo"
+              >
+              <img
+                src="https://retsimages.s3.amazonaws.com/29/A10388229_9.jpg"
+                alt="Faena House photo 2"
+                class="gallery__photo"
+              >
+            </div>
+            <p class="gallery__object-descr">
+              <big class="bright-emphasis">Arvida Dr </big>/ Coral Gables, FL 33156
+            </p>
+          </section>
+          <section class="gallery__object">
+            <div class="gallery__images-row gallery__images-row--1-2">
+              <img
+                src="https://retsimages.s3.amazonaws.com/52/A10890752_7.jpg"
+                alt="Faena House photo 1"
+                class="gallery__photo"
+              >
+              <img
+                src="https://retsimages.s3.amazonaws.com/52/A10890752_2.jpg"
+                alt="Faena House photo 2"
+                class="gallery__photo"
+              >
+            </div>
+            <p class="gallery__object-descr">
+              <big class="bright-emphasis">Mediterranean Villa </big>/ Miami, FL 33135
+            </p>
+          </section>
+        </div>
+
+        <label class="gallery__button" for="photosHider">
+          View all
+        </label>
+      </article>
+
+      <article class="consultation" id="consultation">
+        <h2 class="title-with-crown">
+          Consultation
+        </h2>
+        <div class="consultation__content-box">
+          <section class="consultation__descr-box">
+            <h3 class="consultation__title">
+              <span class="bright-emphasis">How long have you do<br>yourself a gift?</span>
+            </h3>
+            <p class="consultation__text">
+              The procedural change mezzo forte starts izoritmichesky pickup at these moments stop LA Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
+            </p>
+          </section>
+          <form
+            action=""
+            method="get"
+            class="form"
```

Comment:
what kind of form? specify the type of it

Reply:
in this site there is one kind of form and the only one. Form when user takes data to sellsman call him.

Reply:
anyway it would be great if you create more specific name

Reply:
I think it will just make name longer. It is not good.
And I cant imagine what kind of form can be on this site

Reply:
registration-form for example

Reply:
you force me to make more excessfull text, but ok